### ISSUE TITLE
Strange colors on android simulator

### ISSUE THREAD

**Issue by michelmoorlag** [Brainflab/XamaRed.Forms.Svg#2]
On real android and ios devices and on the ios simulator on the Mac the svg's are shown correct. On android simulators the svg's get strange colors. For example if I use a 5.2" Marsmallow (6.0.0) XXHDPI Phone (Android 6.0 -API23) as simulator and I use the svglogo.svg provided in the samples I get a real weird result. See attached screen. Colors also change per debug session. Sometimes it's shades of blue, sometimes shades of pink.
![svglogoandroidsim](https://cloud.githubusercontent.com/assets/14913939/25705016/12b083be-30dc-11e7-80c5-d80c23b97e0a.JPG)

**Comment by Brainflab** [comment]
I can reproduce this issue on VS android simulators.
I'll investigate but I think this might be related to SkiaSharp.

**Comment by Brainflab** [comment]
Are you using a Hyper-V emulator (such as Visual Studio emulators) ?
It seems related to ANGLE & OpenGL rendering : https://blogs.msdn.microsoft.com/visualstudioalm/2014/11/12/introducing-visual-studios-emulator-for-android/
It works fine on AVD emulators, but this workaround is not good enough since Hyper-V is now the recommended way to go on Windows.
I currently don't have a proper solution for this issue, sorry

**Comment by michelmoorlag** [comment]
Yes I am using a Hyper-V emulator indeed. For testing it is a bit annoying but as long as real devices work fine it is not a major issue for me. Maybe it will be fixed on the simulator side in the future.

**Comment by michelmoorlag** [comment]
It is related to Skiashap version 1.57.1. I am using an other plugin to show a svg. With skiasharp 1.57.1 I have exactly the same issue. When I downgrade to version 1.56.1 it all works fine!
